frontend/src/metabase/dashboard/components/DashCard/DashCardVisualization.tsx

```diff
@@ -351,6 +397,10 @@ export function DashCardVisualization({
       );
     }
 
+    // We only show the titleMenuItems if the card has no title.
+    const settings = getComputedSettingsForSeries(series) as any;
+    const title = settings["card.title"] ?? series?.[0].card.name ?? "";
```

Comment:
Can we use the `ComputedVisualizationSettings` type from `"metabase/visualizations/types"` here?